seqr/views/apis/summary_data_api.py

```diff
@@ -172,6 +183,140 @@ def bulk_update_family_analysed_by(request):
     })
 
 
+def _load_aip_data(data, user):
```

Comment:
on what cadence will AIP data be loaded? are analysts the users that do that?

Reply:
No idea! Realistically, its going to be Lynn doing the loading for now, but not sure what that looks like in the future. The ticket specified that this upload should be done using the existing "External Analysis" UI, which is restricted to analysts, so for now thats what we are going with